Source/Python/WarpXWrappers.cpp

```diff
@@ -392,10 +404,12 @@ extern "C"
     WARPX_GET_LOVECTS_PML(warpx_getCurrentDensityCPLoVects_PML, Getj_cp)
     WARPX_GET_LOVECTS_PML(warpx_getCurrentDensityFPLoVects_PML, Getj_fp)
 
-    amrex::ParticleReal** warpx_getParticleStructs(int speciesnumber, int lev,
-                                      int* num_tiles, int** particles_per_tile) {
+    amrex::ParticleReal** warpx_getParticleStructs(
+            const char* char_species_name, int lev,
+            int* num_tiles, int** particles_per_tile) {
```

Comment:
I think it might better to match the styling that WarpX already uses for these functions, ie, the arguments begin to be defined on the same line as the function definition. Additionally, I think it helps with readability, since the indenting of the arguments in this function isn't consistent with the indenting of arguments in later functions, and the indenting makes it easier to confuse the argument definition with the function's contents.

Reply:
Same comment as above.